content/guides/use-case/genai-video-bot/_index.md

```diff
@@ -0,0 +1,243 @@
+   $ docker compose up --build
+   ```
+   Docker Compose builds and runs the application based on the services defined
+   in the `docker-compose.yaml` file. When the application is running, you'll
+   see the logs of 2 services in the terminal.
+
+   In the logs, you'll see the services are exposed on ports `8503` and `8504`.
+   The two services are complimentary to each other.
+
+   The `yt-whisper` service is running on port `8503`. This service feeds the
+   Pinecone database with videos that you want to archive in your knowledge
+   database. The following section explores this service.
+
+## Using the yt-whisper service
+
+The yt-whisper service is YouTube video processing service that uses the OpenAI
+Whisper model to generate transcriptions of videos and stores them in a Pinecone
+database. The following steps show how to use the service.
+
+1. Open a browser and access the yt-whisper service at [http://localhost:8503](http://localhost:8503).
+2. Once the application appears, in **Youtube URL** specify a Youtube video URL
+   and select **Submit**. The following example uses
+   [https://www.youtube.com/watch?v=yaQZFhrW0fU](https://www.youtube.com/watch?v=yaQZFhrW0fU).
+
+   ![Submitting a video in the yt-whisper service](images/yt-whisper.webp)
+
+   The yt-whisper service downloads the audio of the video, uses Whisper to
+   transcribe it into a WebVTT (`*.vtt`) format (which you can download), then
+   uses the text-embedding-3-small model to create embeddings, and finally
+   uploads those embeddings in to the Pinecone database.
+
+   After processing the video, a video list appears in the web app that informs
+   you which videos have been indexed in Pinecone. It also provides a button to
+   download the transcript.
+
+   ![A processed video in the yt-whisper service](images/yt-whisper-2.webp)
+
+   You can now access the dockerbot service on port `8504` and ask questions
+   about the videos.
+
+## Using the dockerbot service
+
+The dockerbot service is a question-answering service that leverages both the
+Pinecone database and an AI model to provide responses. The following steps show
+how to use the service.
+
+> **Note**
+>
+> You must process at least one video via the
+> [yt-whisper service](#using-the-yt-whisper-service) before using
+> the dockerbot service.
+
+1. Open a browser and access the service at
+   [http://localhost:8504](http://localhost:8504).
+
+2. In the **What do you want to know about your videos?** text box, ask the
+   Dockerbot a question about a video that was processed by the yt-whisper
+   service. The following example asks the question, "What is a sugar cookie?".
+   The answer to that question exists in the video processed in the previous
+   example,
+   [https://www.youtube.com/watch?v=yaQZFhrW0fU](https://www.youtube.com/watch?v=yaQZFhrW0fU).
+   
+   ![Asking a question to the Dockerbot](images/bot.webp)
+   
+   As you can see in the previous image, the Dockerbot answers the question and
```

Comment:
**Thought:** I wonder if the wording for this should be updated as some users might not view the image and may be using a screenreader, so they'll be relying on this paragraph to explain the example: 
```suggestion
   In this example, the Dockerbot answers the question and
```